src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -5299,7 +5300,7 @@ namespace CondenserLoopTowers {
         Par(4) = Twb;                // inlet air wet-bulb temperature [C]
         Real64 Tr;                   // range temperature which results in an energy balance
         auto f = std::bind(&CoolingTower::residualTr, this, std::placeholders::_1, std::placeholders::_2);
-        General::SolveRoot(Acc, MaxIte, SolFla, Tr, f, 0.001, towers(this->VSTower).MaxRangeTemp, Par);
+        General::SolveRoot(Acc, MaxIte, SolFla, Tr, f, 0.001, VSTowerMaxRangeTemp, Par);
```

Comment:
@Nigusse Is this overriding the range for all tower types?
Rather than overriding this here, why not set `towers(this->VSTower).MaxRangeTemp` up in `GetTowerInput`, [here](https://github.com/NREL/EnergyPlus/blob/c9d665d7d94dda642053ae68f14c734b0a502bbd/src/EnergyPlus/CondenserLoopTowers.cc#L1164).

Reply:
@mjwitte Below is my suggestion for your consideration. First, this change impacts the toolcool VS cooling tower model only; second, this model will be able to check and throw warning if the requested tower range is outside the VS cooltool tower range maximum limit.

```
        VSTowerMaxRangeTemp = towers(this->VSTower).MaxRangeTemp;
        if (this->TowerModelType == CoolToolsXFModel) {
            VSTowerMaxRangeTemp = 22.2222; // this relaxes the solution domain
        }
```

Reply:
Done. Thanks!